File: files/200-299/205/CH-205.py
import pandas as pd

path = "CH-205 Missing Char.xlsx"
input = pd.read_excel(path, usecols="B", skiprows=1, nrows=6)
test = pd.read_excel(path, usecols="D", skiprows=1, nrows=6)

def replace_slashes_with_indices(word):
    return ''.join(str(i + 1) if char == '/' else char for i, char in enumerate(word))

input['Modified'] = input.iloc[:, 0].apply(replace_slashes_with_indices)

File: files/200-299/205/CH-205.R
library(tidyverse)
library(readxl)

path = "files/CH-205 Missing Char.xlsx"
input = read_excel(path, range = "B2:B7")
test  = read_excel(path, range = "D2:D7")

result = input %>%
  mutate(w_index = row_number()) %>%
  separate_rows(ID, sep = "") %>%
  filter(ID != "") %>%
  mutate(l_index = row_number(), .by = w_index) %>%
  mutate(ID = ifelse(ID == '/', l_index, ID)) %>%
  summarise(ID = paste(ID, collapse = ""), .by = w_index) %>%
  select(-w_index)